src/Avalonia.Themes.Default/NotificationCard.xaml

```diff
@@ -0,0 +1,100 @@
+                            BorderBrush="{TemplateBinding BorderBrush}" 
+                            BorderThickness="{TemplateBinding BorderThickness}"
+                            Margin="8,8,0,0">
+                        <ContentControl MinHeight="150" Content="{TemplateBinding Content}" />
+                    </Border>
+                </LayoutTransformControl>
+            </ControlTemplate>
+        </Setter>
+
+        <Style.Animations>
+            <Animation Duration="0:0:0.45" Easing="QuadraticEaseIn" FillMode="Forward">
+                <KeyFrame Cue="0%">
+                    <Setter Property="Opacity" Value="0"/>
+                    <Setter Property="TranslateTransform.Y" Value="20"/>
+                    <Setter Property="ScaleTransform.ScaleX" Value="0.85"/>
+                    <Setter Property="ScaleTransform.ScaleY" Value="0.85"/>
+                </KeyFrame>
+                <KeyFrame Cue="30%">
+                    <Setter Property="TranslateTransform.Y" Value="-20"/>
+                </KeyFrame>
+                <KeyFrame Cue="100%">
+                    <Setter Property="Opacity" Value="1"/>
+                    <Setter Property="TranslateTransform.Y" Value="0"/>
+                    <Setter Property="ScaleTransform.ScaleX" Value="1"/>
+                    <Setter Property="ScaleTransform.ScaleY" Value="1"/>
+                </KeyFrame>
+            </Animation>
+        </Style.Animations>
+    </Style>
+
+    <Style Selector="NotificationCard[IsClosing=true] /template/ LayoutTransformControl#PART_LayoutTransformControl">
+        <Setter Property="RenderTransformOrigin" Value="50%,0%"/>
+        <Style.Animations>
+            <Animation Duration="0:0:0.75" Easing="QuadraticEaseOut" FillMode="Forward">
+                <KeyFrame Cue="0%">
+                    <Setter Property="TranslateTransform.X" Value="0"/>
+                    <Setter Property="ScaleTransform.ScaleY" Value="1"/>
+                </KeyFrame>
+                <KeyFrame Cue="70%">
+                    <Setter Property="TranslateTransform.X" Value="800"/>
+                    <Setter Property="ScaleTransform.ScaleY" Value="1"/>
+                </KeyFrame>
+                <KeyFrame Cue="100%">
+                    <Setter Property="ScaleTransform.ScaleY" Value="0"/>
+                    <Setter Property="TranslateTransform.X" Value="800"/>
+                </KeyFrame>
+            </Animation>
+        </Style.Animations>
+    </Style>
+
+    <Style Selector="NotificationCard[IsClosing=true]">
+        <Style.Animations>
+            <Animation Duration="0:0:1.25" Easing="QuadraticEaseOut" FillMode="Forward">
+                <KeyFrame Cue="100%">
+                    <Setter Property="IsClosed" Value="True"/>
+                </KeyFrame>
+            </Animation>
+        </Style.Animations>
+    </Style>
+
+    <Style Selector="NotificationCard /template/ Button#PART_CloseButton">
```

Comment:
There doesn't seem to be a `PART_CloseButton` in the `NotificationCard` template.

Reply:
removed, since we dont need a close button.